google/cloud/storage/tests/curl_request_integration_test.cc

```diff
@@ -366,9 +366,10 @@ TEST(CurlRequestTest, WellKnownQueryParametersMultiple) {
 
 class MockLogBackend : public google::cloud::LogBackend {
  public:
-  void Process(google::cloud::LogRecord const& lr) override {
-    ProcessWithOwnership(lr);
-  }
+  // TODO(#3983) clang-tidy and clang-3.8 conflict about annotating
+  // `Process` as `override`. See the issue for details.
+  // NOLINTNEXTLINE(modernize-use-override)
+  void Process(google::cloud::LogRecord const& lr) { ProcessWithOwnership(lr); }
   MOCK_METHOD1(ProcessWithOwnership, void(google::cloud::LogRecord));
```

Comment:
I think you may be able to fix both warnings and avoid the NOLINT if we change `MOCK_METHOD1` a  `MOCK_METHOD` call:

```cc
  MOCK_METHOD(void, ProcessWithOwnership, (google::cloud::LogRecord), (override));
```